docs/modules/ROOT/pages/references/cp_node_scaling.adoc

```diff
@@ -0,0 +1,51 @@
+= Control Plane Node Scaling
+
+The control plane node resource requirements depent on a number of different metrics.
+CPU requirements generally depend on the ammount on API write requests,
+while memory requirements generally depend on the amount of resource objects (most importantly Pod count) in the cluster.
+
+To avoid cascading failures during a scheduled node reboot or an unexpected node failure,
+it's recommended to keep the overall CPU and memory usage not significantly higher than 60%.
```

Comment:
```suggestion
To avoid cascading failures during a scheduled node reboot or an unexpected node failure, it's recommended to keep the overall CPU and memory usage not significantly higher than 60%.
```